applications/ConvectionDiffusionApplication/CMakeLists.txt

```diff
@@ -14,6 +14,8 @@ set( KRATOS_CONVECTION_DIFFUSION_APPLICATION_CORE_SOURCES
     ${CMAKE_CURRENT_SOURCE_DIR}/custom_elements/conv_diff_2d.cpp
     ${CMAKE_CURRENT_SOURCE_DIR}/custom_elements/conv_diff_3d.cpp
     ${CMAKE_CURRENT_SOURCE_DIR}/custom_elements/laplacian_element.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/custom_elements/symbolic_quasi_static_eulerian_convection_diffusion_explicit.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/custom_elements/symbolic_dynamic_eulerian_convection_diffusion_explicit.cpp
```

Comment:
I found that these filenames (and class names) are way too long. Considering that we have experienced problems with these I suggest renaming the files. Class names should be renamed accordingly.
`eulerian` specifier is not needed as this is the common framework in convection-diffusion problems.
`_dynamic_` can be specified as `_d_` or `_dyn_` as we do in the fluid
`_quasi_static_` can be specified as `_qs_` as in the fluid elements

Reply:
Ok. I will do the change.

Reply:
OK. Once you change this I will review each file one by one.

Reply:
Done @rubenzorrilla.